libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -713,15 +762,12 @@ string YulUtilFunctions::overflowCheckedSignedExpFunction()
 						// for base * base above is sufficient, since:
 						// |power| <= base (proof by induction) and thus:
 						// |power * base| <= base * base <= max <= |min|
```

Comment:
```suggestion
						// |power * base| <= base * base <= max <= |min| (for signed)
```
Maybe it's better to add a comment since the part about `|min|` is only true for signed.